python/pyspark/sql/connect/dataframe.py

```diff
@@ -1775,6 +1775,10 @@ def _to_table(self) -> Tuple["pa.Table", Optional[StructType]]:
         assert table is not None
         return (table, schema)
 
+    def _toArrow(self) -> "pa.Table":
+        table = self._to_table()[0]
```

Comment:
nit: Using this notion, makes the code a bit more readable and makes sure it fails if _to_table ever changes the tuple size, and thus behavior.

```suggestion
        table, _ = self._to_table()
```